sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -2447,6 +2447,23 @@ await foreach (BlobItem blobItem in test.Container.GetBlobsAsync(states: BlobSta
             Assert.IsTrue(blobItems[0].HasVersionsOnly);
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2021_02_12)]
+        public async Task ListBlobsFlatSegmentAsync_EncodedBlobName()
+        {
+            // Arrange
+            await using DisposingContainer test = await GetTestContainerAsync();
+            string blobName = "dir1/dir2/file\uFFFF.blob";
```

Comment:
should we add more characters like we did for other encoding tests? (umlauts and so on)